tests_e2e/orchestrator/scripts/install-agent

```diff
@@ -72,6 +72,14 @@ else
 fi
 echo "Service name: $service_name"
 
+#
```

Comment:
There was a race condition between get-agent-path and the agent service restart a few lines below (get-agent-path uses the pid file of the agent in its logic) so I moved the initialization here, before we stop and restart the service